src/en/foundations/colour/usage.md

```diff
@@ -12,31 +12,66 @@ eleventyNavigation:
   alt: This is an image of the foundation
 permalink: /en/foundations/colour/
 translationKey: "colour"
-tags: ['colourEN', 'usage']
+tags: ["colourEN", "usage"]
+type: color
+headers:
+  color-preview: ""
+  token-name: Token name
+  value: Hex
+  rgb: RGB
+  use-case: Use case
```

Comment:
the keys are used to display a type of td in the `{% case %}`. The values are the labels and can be localized